api/src/main/java/org/openmrs/FieldType.java

```diff
@@ -9,20 +9,49 @@
  */
 package org.openmrs;
 
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.Parameter;
 import org.hibernate.envers.Audited;
 
+import javax.persistence.AttributeOverride;
+import javax.persistence.AttributeOverrides;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
 /**
  * FieldType
  */
+@Entity
+@Table(name = "field_type")
 @Audited
+@AttributeOverrides(value = {
+	@AttributeOverride(name = "name", column = @Column(name = "name", length = 50, nullable = false)),
+	@AttributeOverride(name = "retired", column = @Column(name = "retired", columnDefinition = "boolean default false"))
```

Comment:
Why use "boolean default false" here?'

Reply:
we should set the `retired` column default value as `false` ?

Reply:
In BaseOpenmrsMetadata default value for retired is set to false.

https://github.com/openmrs/openmrs-core/blob/335c2b5186086f8a3b3d1d353bb1d06d23a20dcd/api/src/main/java/org/openmrs/BaseOpenmrsMetadata.java#L58-L60

Reply:
without including `columnDefinition = "boolean default false"`, `HL7Service, OrderEntryIntergationTest, ` and `MedicationDispenseServiceTest` tests are getting failed.
 

Caused by:` org.h2.jdbc.JdbcSQLIntegrityConstraintViolationException: NULL not allowed for column "RETIRED"; SQL statement:
insert into FIELD_TYPE (FIELD_TYPE_ID, NAME, IS_SET, CREATOR, DATE_CREATED, UUID) values (?, ?, ?, ?, ?, ?) [23502-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:459)`

Reply:
@dkayiwa, @ibacher what are your thoughts on this?

Reply:
I do not like the column definition value of boolean default false
Can we think of an alternative?